{
    "id": "CVE-2017-6301",
    "product": "n/a",
    "version": "n/a ",
    "vulnerability": [
        "n/a"
    ],
    "description": "An issue was discovered in ytnef before 1.9.1. This is related to a patch described as \"4 of 9. Out of Bounds Reads.\"",
    "poc": {
        "reference": [
            "https://github.com/Yeraze/ytnef/pull/27"
        ],
        "github": [
            "No PoCs found on GitHub currently."
        ]
    }
}